tools/nhlt-parse.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+#
+# usage: nhlt-parse.py <nhtl-dump>
+#
+# examples:
+#   cat /sys/firmware/acpi/tables/NHLT >nhtl.bin
+#   ./nhtl-parse.py nhtl.bin
+#
+# references:
+#  - https://01.org/sites/default/files/595976_intel_sst_nhlt.pdf
+
+from collections import namedtuple
+import struct
+import sys
+
+if len(sys.argv) == 1:
+    infile = '/sys/firmware/acpi/tables/NHLT'
+else:
```

Comment:
could you use some our existing arguments parsing style to add more extendable options here? like `-h` and `-o INFILE`

Reply:
`argparse` is great but a bit overkill for only one argument. Later?